pom.xml

```
@@ -23,14 +23,15 @@
     <properties>
         <commons.lang.version>3.12.0</commons.lang.version>
 
-        <mule.http.connector.version>1.7.3</mule.http.connector.version>
-        <muleSdkApiVersion>0.3.0</muleSdkApiVersion>
+        <mule.http.connector.version>1.9.0</mule.http.connector.version>
+        <muleSdkApiVersion>0.7.0</muleSdkApiVersion>
         <soap.engine.version>2.0.0-SNAPSHOT</soap.engine.version>
-        <wsdl.parser.version>1.5.0-SNAPSHOT</wsdl.parser.version>
+        <wsdl.parser.version>1.4.7</wsdl.parser.version>
```

Is this version of mule-wsdl-parser java 17 compatible?

---

No, it's not, it should be 1.4.8-SNAPSHOT for this SNAPSHOT release
```suggestion
        <wsdl.parser.version>1.4.8-SNAPSHOT</wsdl.parser.version>
```

---

We donn't know, there's still no wsdl parser ready. As this is a snapshot it's not a blocker